src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -249,6 +249,13 @@
         </property>
        </widget>
       </item>
+      <item row="4" column="0" colspan="2">
+       <widget class="QCheckBox" name="checkBoxEditMetadataSelectedClicked">
+        <property name="text">
+         <string>Edit metadata after clicking selected track</string>
```

Comment:
How about:  "Allow editing of metadata by clicking selected track"

I wonder if this is a good candidate for an inverted bool, because the major use-case is "locking" the library so it can't accidentally be edited: "Prevent editing of metadata by clicking selected track"

Reply:
Yes, good idea.